app-i18n/fcitx-configtool/fcitx-configtool-5.1.6-r1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="fcitx5-configtool"
+
+inherit cmake unpacker
+
+DESCRIPTION="Configuration module for Fcitx"
+HOMEPAGE="https://fcitx-im.org/ https://github.com/fcitx/fcitx5-configtool"
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_PN}-${PV}.tar.zst -> ${P}.tar.zst"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+LICENSE="GPL-2+"
+SLOT="5"
+KEYWORDS="~amd64"
+IUSE="kcm +config-qt +qt5 qt6 test"
```

Comment:
Umm, looking at the code, this application can be build using qt5 xor qt6 (not both), so you want to use the instructions at https://wiki.gentoo.org/wiki/Project:Qt/Policies#Requires_one_of_two_Qt_versions

So it would be just `IUSE="qt6"`, and you replace `qt5` with `!qt6`